.github/workflows/asset-push.yml

```diff
@@ -29,3 +29,15 @@ jobs:
         run: |
           cd bin/hoist
           cargo run -- upload-all-specs -y --non-interactive --max-concurrent 25 -t ../clover/si-specs
+
+      - name: Update Module Cache in Tools Prod
+        run: |
+          curl -X POST "https://tools.systeminit.com/api/v2/admin/update_module_cache" -H "Authorization: Bearer ${{ env.MY_BEARER_TOKEN }}"
+        env:
+          MY_BEARER_TOKEN: ${{ secrets.SI_PRODUCTION_SAAS_WORKSPACE_API_TOKEN }}
+          
+      - name: Update Module Cache in Prod
+        run: |
+          curl -X POST "https://app.systeminit.com/api/v2/admin/update_module_cache" -H "Authorization: Bearer ${{ env.MY_BEARER_TOKEN }}"
```

Comment:
Does this return after success or fire and let it run in the background? I ask because I imagine we want to avoid pushing to prod if there are issues with the generated modules in tools.

Reply:
This is an Async API - it's a fire and forget - even in the UI

Reply:
I guess I worry about pushing a bad update to prod then. We don't really have a way to rollback changes here. Let's talk about it